### ISSUE TITLE
Replace JS bot by neural bot

### ISSUE THREAD

**Issue by laerling** [hkgit03/paperclipmaximizermaximizer#4]
- What is the genome? Initial synapse and neuron values? Hyperparameters? Both?
- Sigmoid is so 2016, use [ReLU](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)). Less waste of computing power and no vanishing gradience.
- Idea: Decrease learning rate over time, but increase it again each time new buttons and indicators occur.
- Convolution?
- Talk to jeff again.
